feature-pack/src/main/resources/modules/system/layers/base/javax/ejb/api/main/module.xml

```diff
@@ -25,11 +25,11 @@
 <module xmlns="urn:jboss:module:1.3" name="javax.ejb.api">
     <dependencies>
         <!-- This dep is for javax.naming -->
-        <module name="javax.api" export="true"/>
-        <module name="javax.transaction.api" export="true"/>
-        <module name="javax.xml.rpc.api" export="true"/>
```

Comment:
Isn't that really a problem with the exclusion functionality? If I want to exclude a module I would expect it to exclude any transitive ones as well.